src/SFA.DAS.DownloadService.Api/Controllers/AparController.cs

```diff
@@ -86,7 +84,7 @@ public async Task<IActionResult> Get(int ukprn)
 
             OrganisationModel model = await _roatpApiClient.GetRoatpSummaryByUkprn(ukprn);
 
-            if (model == null || model.Status == OrganisationStatus.Removed)
+            if (model == null || (int)model.Status == 0)
```

Comment:
We can check this directly using enum? if not instead of using magic number, constant it so it has a meaning